Temporarily fix for #101

antechrestos opened the pull request:
Disable urls invoked for dashboard

Commit:
Disable authentication for dashboard (temporarily fix for #101)

arnaudruffin commented on src/main/java/org/cloudfoundry/autosleep/servicebroker/service/AutoSleepServiceInstanceService.java:
```diff
@@ -79,7 +79,7 @@ public ServiceInstance createServiceInstance(CreateServiceInstanceRequest reques
             serviceInstance = new AutosleepServiceInstance(request);
             if (deployment != null) {
-                serviceInstance.withDashboardUrl(deployment.getFirstUri() + Config.Path.dashboardPrefix + serviceId);
+                serviceInstance.withDashboardUrl(deployment.getFirstUri() + Config.Path.dashboardContext + serviceId);
```
Aren't we missing a slash over here?

gberche-orange commented on src/main/java/org/cloudfoundry/autosleep/config/Config.java:
```diff
@@ -10,7 +10,10 @@
 
     interface Path {
-        String dashboardPrefix = "/dashboard/";
+        String apiContext = "/api";
```
Would it make sense to use the common UPPER_CASE naming convention for these constants ?

Commit:
Modifications:
- correction of a missing / in dashboard url
- Modifications of constant name convention
- isolation of javascript code for dashboard page
- not found returned if service instance not found
